lib/peer.js

```diff
@@ -327,7 +327,8 @@ class Peer extends EventEmitter {
 
   handshake () {
     const opts = {
-      dht: this.swarm.private ? false : !!this.swarm.client.dht
+      dht: this.swarm.private ? false : !!this.swarm.client.dht,
+      fast: true
```

Comment:
there is a bit in the reserved handshake field used to indicate whether the peer supports fast extension or not. Perhaps I'm misreading this, but it looks like you just assume the peer supports the FAST extension.

http://bittorrent.org/beps/bep_0006.html

> reserved[7] |= 0x04

Reply:
It only enables Fast if both peers support it, this is actually done in the dependency [webtorrent/bittorrent-protocol](https://github.com/webtorrent/bittorrent-protocol/blob/94f72e96e55a90a261c064495dcbda13c653a0cd/index.js#L752).